networking/synthetic/stac-test.yaml

```diff
@@ -8,9 +8,21 @@
     
   - name: Copy Dockerfile to producer and consumer
     copy: src=stac/Dockerfile dest=/tmp/stac/Dockerfile
+
+  - name: Copy onload rpms to producer and consumer
+    copy: src=stac/openonload-201606_u1.2-1.el7.x86_64.rpm dest=/tmp/stac/openonload-201606_u1.2-1.el7.x86_64.rpm
```

Comment:
hard coded versions going to bite us; probably should have this filename as an object in the ansible inventory